scaling.md

```diff
@@ -0,0 +1,78 @@
+# Scaling Application Service Adapter
+
+This topic provides guidance on scaling Application Service Adapter components
+for performance or availability.
+
+* [Notes on system performance](#system-performance)
+* [Scaling the Application Service Adapter controller managers](#controllers)
+* [Scaling the Application Service Adapter API](#api)
+
+## <a id="system-performance"></a>Notes on system performance
+
+The Application Service Adapter runs on a Kubernetes cluster and makes
+extensive use of the Kubernetes API to store and manage resources. For these
+reasons, the performance of the Application Service Adapter is very dependent on
+the underlying Kubernetes control plane. If you observe slow performance or
+timeouts when using the Application Service Adapter, a good first step is to
+ensure that the control plane for your Kubernetes cluster is properly resourced.
+
+For managed clusters that come from public-cloud IaaSes, you generally don't
```

Comment:
This overall section around ensuring the K8s control plane has sufficient resources is very good to call out. 👍 

> you generally don't need to be concerned about scaling the control plane

For this piece I might rephrase it around something like "you generally won't have direct control over how the control plane is provisioned" or something. Cause you might actually be concerned (like how we've seen kapp knock over small GKE cluster when their control plane couldn't handle it), but you just won't have direct control to fix it. I dunno, a bit of a nitpick, but something to consider.

Reply:
done.